Core/Steps/SubSteps/AddFixVersionsForNewReleaseBranchSubStep.cs

```diff
@@ -0,0 +1,46 @@
+using Remotion.ReleaseProcessAutomation.Extensions;
+using Remotion.ReleaseProcessAutomation.Jira;
+using Remotion.ReleaseProcessAutomation.Jira.ServiceFacadeImplementations;
+using Remotion.ReleaseProcessAutomation.ReadInput;
+using Remotion.ReleaseProcessAutomation.SemanticVersioning;
+using Spectre.Console;
+
+namespace Remotion.ReleaseProcessAutomation.Steps.SubSteps;
+
+public interface IAddFixVersionsForNewReleaseBranchSubStep
+{
+  public void Execute (SemanticVersion currentVersion, SemanticVersion nextJiraVersion);
```

Comment:
Interface methods never have an access modifier as they are always `public`

```suggestion
  void Execute (SemanticVersion currentVersion, SemanticVersion nextJiraVersion);
```